tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -1327,3 +1339,163 @@ Feature: get users
       | Space Admin |
       | User        |
       | User Light  |
+
+
+  Scenario Outline: non-admin user searches other users by display name
+    Given the administrator has assigned the role "<role>" to user "Alice" using the Graph API
+    And the administrator has assigned the role "<userRole>" to user "Brian" using the Graph API
```

Comment:
```suggestion
    Given the administrator has assigned the role "<role>" to user "Alice" using the Graph API
```
What is the purpose of assigning roles to users? If searching has to be done with different roles then why assign roles to two users? Retext the scenario description.